scripts/rspec/rspec-templates/VbNetRuleTemplate.cs

```diff
@@ -35,8 +35,19 @@ namespace SonarAnalyzer.Rules.VisualBasic
     [Rule(DiagnosticId)]
     public sealed class $DiagnosticClassName$ : $DiagnosticClassName$Base
     {
-        private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(rule);
+        private static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
```

Comment:
This should be in the base class. Constructor should be added instead.

Reply:
Later edit: There might be more complex logic needed for VB & C# to distinguish between standalone rule vs. adding 2nd rule.
Something like:
* new Rule => use this file and place members snippet here (the long one, with rule ID and message format)
* adding 2nd rule => use this file and place constructor snippet here

Those inner snippets would be common for C# & VB & base class for the fields (not constructor)